Source/ForecastDeskbarView.cpp

```diff
@@ -156,5 +156,5 @@ extern "C" BView* instantiate_deskbar_item(float maxWidth, float maxHeight);
 BView*
 instantiate_deskbar_item(float maxWidth, float maxHeight)
 {	
-	return new ForecastDeskbarView(BRect(0, 0, maxWidth - 1, maxHeight - 1));
```

Comment:
this should use something like `std::min(maxWidth, maxHeight)`, to compute the largest square that can fit in the space allowed by DeskBar.

This makes the replicant usable no matter what layout DeskBar decides to use in future versions.

Reply:
Thanks @pulkomandy, I've updated the code